src/Arcus.EventGrid.Publishing/EventGridPublisherBuilderResult.cs

```diff
@@ -39,18 +40,23 @@ public EventGridPublisherBuilderResult(string topicEndpoint, string authenticati
         /// <exception cref="ArgumentException">The topic endpoint must not be empty and is required</exception>
         /// <exception cref="ArgumentException">The authentication key must not be empty and is required</exception>
         /// <exception cref="ArgumentNullException">The resilient policy is required</exception>
+        /// <exception cref="UriFormatException">The topic endpoint must be a HTTP endpoint.</exception>
         public EventGridPublisherBuilderResult(
```

Comment:
Given the class is internal we might want to change this ctor to internal?

Reply:
Yes!